frontend/public/components/volumes-table.tsx

```diff
@@ -113,21 +111,17 @@ const VolumesTableHeader = () => {
 VolumesTableHeader.displayName = 'VolumesTableHeader';
 
 const VolumesTableRow = ({obj: volume, index, key, style}) => {
-  const type = _.get(getVolumeType(volume.volumeDetail), 'id', '');
-  const loc = getVolumeLocation(volume.volumeDetail);
-  const name = volume.name;
-  const permission = volume.readOnly ? 'Read-only' : 'Read/Write';
-  const { resource } = volume;
+  const { name, resource, readOnly, mountPath, volumeDetail } = volume;
+  const permission = readOnly ? 'Read-only' : 'Read/Write';
   const pod: PodTemplate = getPodTemplate(resource);
 
   return (
-    <TableRow id={`${name}-${volume.mountPath}`} index={index} trKey={key} style={style}>
+    <TableRow id={`${name}-${mountPath}`} index={index} trKey={key} style={style}>
       <TableData className={volumeRoColumnClasses[0]}>{name}</TableData>
       <TableData className={classNames(volumeRoColumnClasses[1], 'co-break-word')}>{volume.mountPath}</TableData>
       <TableData className={volumeRoColumnClasses[2]}>{volume.subPath}</TableData>
```

Comment:
nit from my original code: `volumeRoColumnClasses` is typo'd everywhere and we could do a replace all with the correct spelling `volumeRowColumnClasses`